CHANGELOG.md

```diff
@@ -22,6 +22,10 @@ Versioning](https://semver.org/spec/v2.0.0.html) for `libnopegl`.
   v2023.2, this is changed back in this release
 - `SceneCfg` and `SceneInfo` now use the `Backend` enum instead of a string
 
+### Removed
+- `%s_dimensions` uniform for 2D array and 3D images/textures, users must use
+  textureSize()/imageSize() instead
```

Comment:
Please state that it is still available for 2D textures, whether they're accelerated or not

Reply:
Done.